app/views/layouts/_alert.html.haml

```diff
@@ -1,5 +1,6 @@
 .alert-box{ class: "#{type}" }
   = message
   - if local_assigns[:unconfirmed]
-    %a{ "data-action": "login-modal#resend_confirmation", "data-tab": local_assigns[:tab] }
+    - params = { spree_user: { email: email }, tab: local_assigns[:tab] }
+    = link_to spree_user_confirmation_path(params), params:, data: { turbo_method: :post } do
```

Comment:
Why is `params` repeated on this line? 
I think it should only be needed the first time. If that's the case, perhaps it can be included in one line. Does this work?
```suggestion
    = link_to spree_user_confirmation_path(spree_user: { email: }, tab: local_assigns[:tab]), data: { turbo_method: :post } do
```

Reply:
I have to admit I don't exactly see how this works, but can see that it mimics the behaviour of the `resend_confirmation` javascript that it replaces. 

~~`±` By the way, [`resend_confirmation`](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/app/webpacker/controllers/login_modal_controller.js#L69-L80) can be removed if it's now unused.~~ It's removed in a later commit

Reply:
I am unsure where it's removed.
However, this is used when a user attempts to log in without confirming their email address.
Then this option to resend the confirmation email is shown.

Reply:
> I am unsure where it's removed.

The JS function `login-modal#resend_confirmation` is removed in the commit "Handle user confirmation with turbo stream" here: https://github.com/openfoodfoundation/openfoodnetwork/pull/12997#discussion_r1866239031

All good :thu